tools/deploy-calibnet.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+# deploy-calibnet deploys the Payments contract to Filecoin Calibration Testnet
+# Assumption: KEYSTORE, PASSWORD, RPC_URL env vars are set to an appropriate eth keystore path and password
+# and to a valid RPC_URL for the calibnet.
+# Assumption: forge, cast, jq are in the PATH
+# Assumption: called from contracts directory so forge paths work out
+#
+echo "Deploying Payments to calibnet"
+
+if [ -z "$RPC_URL" ]; then
+  echo "Error: RPC_URL is not set"
+  exit 1
+fi
+
+if [ -z "$KEYSTORE" ]; then
+  echo "Error: KEYSTORE is not set"
+  exit 1
+fi
+
+ADDR=$(cast wallet address --keystore "$KEYSTORE" --password "$PASSWORD")
+echo "Deploying Payments from address $ADDR"
+# Parse the output of forge create to extract the contract address
+ 
+NONCE="$(cast nonce --rpc-url "$RPC_URL" "$ADDR")"
+
+echo "Deploying Payments implementation"
+PAYMENTS_IMPLEMENTATION_ADDRESS=$(forge create --rpc-url "$RPC_URL" --keystore "$KEYSTORE" --password "$PASSWORD" --broadcast --nonce $NONCE --chain-id 314159 src/Payments.sol:Payments | grep "Deployed to" | awk '{print $3}')
```

Comment:
Here and throughout you're not using IMPLEMENTATION_PATH but instead hardcoding to src/Payments.sol.  This is a good default but you should check for IMPLEMENTATION_PATH equal to empty string before using it.  The other alternative is to not mention the unused IMPLEMENTATION_PATH var in the readme.